charts/datasaur/values.yaml

```diff
@@ -155,6 +172,8 @@ imageCredentials:
 
 ingress:
   host: null
+  llm:
```

Comment:
maybe we could use terminology llmLabs across the helm chart

so this should be llmLabs.host